pkg/cc/config/config.go

```diff
@@ -100,3 +123,41 @@ func newConfig(reader configReader, overrides ...Overrider) (*Config, error) {
 func jsonDecoderConfig(dc *mapstructure.DecoderConfig) {
 	dc.TagName = "json"
 }
+
+func GetConfigFile() (string, error) {
+	env := os.Getenv("ASERTO_ENV")
+	if env != "" {
+		return env, nil
+	}
+	home, err := os.UserHomeDir()
+	if err != nil {
+		return "", errors.Wrap(err, "failed to determine user home directory")
+	}
+	return filepath.Join(home, ".config", x.AppName, "config.json"), nil
```

Comment:
The current config file is in yaml format. Is there a reason to switch to json?

Reply:
It's always been json https://github.com/aserto-dev/aserto/blob/761d2665737b35506f3340c583e5e2ffb1310dc5/pkg/cc/config/config.go#L34